test/unit/services/test_image.py

```diff
@@ -163,6 +163,45 @@ def test_create_with_invalid_entity_id(self):
         self.check_create_failed_with_exception("Invalid ObjectId value 'invalid-id'")
 
 
+class GetDSL(ImageServiceDSL):
+    """Base class for `get` tests."""
+
+    _image_id: str
```

Comment:
```suggestion
    __obtained_image_id: str
```
To match IMS.

Reply:
Does it make sense for it to be called `_obtained...`? 

Because in the test it's only being used to make-up the query, it's not a result from either the `call `or `mock` functions. It is used in the `check_success` functions to assert that the request was made properly, so if it had to have one of the prefixes, it would arguably be `_expected`?

Reply:
I was thinking of it like
_obtained_image_id - Id of the image being obtained via the get
_expected_image - Expected image returned by the get
_obtained_image - Actual obtained image returned by the get

Its not really the expected one as its the one passed into the 'get' function in the first place.